### PR TITLE
util: check for instances file in app dir

### PR DESCRIPTION
**Pull request by psergee**
It is possible to have an application without init.lua script. Script names can be specified using instances file. So if the directory contains instances file, it must be treated as a multi-instance application.

### CODE REVIEW

File: cli/util/util.go [hunk 1]
```diff
@@ -691,19 +691,22 @@ func IsApp(path string) bool {
...
-		return true
-	}
-
-	if _, err = os.Stat(filepath.Join(path, "init.lua")); err == nil && entry.IsDir() {
-		return true
-	} else if entry.IsDir() {
-		return false
+	if entry.IsDir() {
+		// Check if the directory contains init.lua script or instances.yml file.
+		for _, fileTocheck := range [...]string{"init.lua", "instances.yml"} {
```
Comment (LeonidVas) [inline cli/util/util.go:696]: Maybe also include `instances.yaml`?

File: test/integration/running/multi_inst_app_no_init/ie/mi_app/instances.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+router:
```
Comment (LeonidVas) [inline test/integration/running/multi_inst_app_no_init/ie/mi_app/instances.yml:1]: AFAIU "mi" == "multi instalce", but what is "ie"?

**Review by LeonidVas — APPROVED** [review]
Hi! Thank you for the patch. Generally I'm ok.
If I understand correctly, this change affects the customers, so needs to add a note to CHANGELOG.

File: cli/util/util.go (hunk 1 above)
Comment (psergee) [reply cli/util/util.go]: `.yaml` extension support for instances file is added.

File: test/integration/running/multi_inst_app_no_init/ie/mi_app/instances.yml (hunk 2 above)
Comment (psergee) [reply test/integration/running/multi_inst_app_no_init/ie/mi_app/instances.yml]: Instances enabled. Or Internet Explorer O_o
Renamed.

**Pull request merged**